src/net/chat.rs

```diff
@@ -201,6 +204,14 @@ impl NeighborStats {
         }
     }
 
+    pub fn add_stackerdb_push(&mut self, message_size: u64) -> () {
+        self.stackerdb_push_rx_counts
+            .push_back((get_epoch_time_secs(), message_size));
+        while self.stackerdb_push_rx_counts.len() > NUM_BLOCK_POINTS {
+            self.stackerdb_push_rx_counts.pop_front();
+        }
+    }
```

Comment:
Can you add a rustdoc to this method (that also explains that this method will drop older entries)?

Reply:
Fixed